blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientConfig.scala

```diff
@@ -56,7 +57,8 @@ final case class BlazeClientConfig( // HTTP properties
     // pipeline management
     bufferSize: Int,
     executionContext: ExecutionContext,
-    group: Option[AsynchronousChannelGroup]) {
+    group: Option[AsynchronousChannelGroup],
+    statsPublisher: Option[PoolManagerStats => Unit]) {
```

Comment:
There is a `BlazeClientBuilder` now (I don't think it was there when you started), and the major rationale for that was so we could add future configuration in a binary compatible way.  These new properties should be migrated to that.  But that's where we'd want these.

Reply:
Ah yeah, thanks, I did see that. I notice it is now parameterised on `F` so it should be possible to add side-effecting callbacks to that quite easily.